testsupport/space/space.go

```diff
@@ -66,7 +73,7 @@ func CreateSpaceWithRole(t *testing.T, awaitilities wait.Awaitilities, role stri
 		require.NoError(t, err)
 	}
 
-	return space, signup, spaceBinding
+	return space, user, spaceBinding
```

Comment:
is there any benefit for returning the `user` instead of just the `user.UserSignup` if that's the only thing that used? Alternatively I see that the `user` object also contains the space.

Reply:
The main reason is to retrieve the token. I'd too prefer not to change this method, but I could not find a better approach. Do you have any suggestions for retrieving it?

Reply:
Ok makes sense. I've noticed now that is also called from the test. So never mind. Please disregards my comment.